On riscv64, targets.cc may call exit(1) and thus needs stdlib.h due to:

    #if HWY_COMPILER_MSVC
      abort();  // Compile error without this due to HWY_NORETURN.
    #elif HWY_ARCH_RVV
      exit(1);  // trap/abort just freeze Spike
    #else
      __builtin_trap();
    #endif

diff -upr highway-0.15.0.orig/hwy/targets.cc highway-0.15.0/hwy/targets.cc
--- highway-0.15.0.orig/hwy/targets.cc	2021-11-26 19:09:21.848569597 +0100
+++ highway-0.15.0/hwy/targets.cc	2021-11-26 19:09:51.378719391 +0100
@@ -16,6 +16,7 @@
 
 #include <stdarg.h>
 #include <stdint.h>
+#include <stdlib.h>
 #include <stdio.h>
 
 #include <atomic>
